packages/medusa/src/api/admin/promotions/utils/rule-attributes-map.ts

```diff
@@ -46,7 +46,7 @@ const ruleAttributes = [
   },
 ]
 
-const commonAttributes = [
+const productAttributes = [
```

Comment:
They're not product attributes, but attributes you could use for a `applicationMethodTargetType`, which are items, order, shipping_methods

```suggestion
const itemsAttributes = [
```

And then create a new `ordersAttributes = [...itemsAttributes]` since they'd be the same for now.